.github/workflows/e2e-node.yml

```
@@ -22,50 +22,50 @@ jobs:
-      uses: actions/setup-node@v3
-      with:
-        node-version: ${{ matrix.node-version }}
-        cache: npm
-    - run: npm ci
-    - run: npm run build
-    - run: npm run test:e2e:node
-      env:
-        E2E_TEST_NOTIFICATION_GATEWAY: ${{ secrets.E2E_TEST_NOTIFICATION_GATEWAY }}
-        E2E_TEST_NOTIFICATION_PROTOCOL: ${{ secrets.E2E_TEST_NOTIFICATION_PROTOCOL }}
-        E2E_TEST_IDP: ${{ secrets.E2E_TEST_IDP }}
-        E2E_TEST_CLIENT_ID: ${{ secrets.E2E_TEST_CLIENT_ID }}
-        E2E_TEST_CLIENT_SECRET: ${{ secrets.E2E_TEST_CLIENT_SECRET }}
-        E2E_TEST_ENVIRONMENT: ${{ matrix.environment-name }}
-    - name: Add status check with successful test result
-      id: set-status-check-success
-      uses: octokit/request-action@v2.1.0
-      with:
-        route: POST /repos/:repository/statuses/:ref
-        repository: ${{ github.repository }}
-        ref: ${{ github.sha }}
-        state: success
-        context: 'End-to-end tests (node)'
-        description: 'Node.js notification end-to-end tests'
-        target_url: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}?check_suite_focus=true
-        # The deployment runs in parallel with CI, so status checks will never have succeeded yet:
-        required_contexts: "[]"
-      env:
-        GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
-    - name: Add status check with failed test result
-      id: set-status-check-failure
-      if: failure()
-      uses: octokit/request-action@v2.1.0
-      with:
-        route: POST /repos/:repository/statuses/:ref
-        repoitory: ${{ github.repository }}
-        ref: ${{ github.sha }}
-        state: failure
-        context: 'End-to-end tests (node)'
-        description: 'Node.js notification end-to-end tests'
-        target_url: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}?check_suite_focus=true
-        # The deployment runs in parallel with CI, so status checks will never have succeeded yet:
-        required_contexts: "[]"
-      env:
-        GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+      - uses: actions/checkout@v3
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: npm
+      - run: npm ci
+      - run: npm run build
+      - run: npm run test:e2e:node
+        env:
+          E2E_TEST_NOTIFICATION_GATEWAY: ${{ secrets.E2E_TEST_NOTIFICATION_GATEWAY }}
+          E2E_TEST_NOTIFICATION_PROTOCOL: ${{ secrets.E2E_TEST_NOTIFICATION_PROTOCOL }}
+          E2E_TEST_IDP: ${{ secrets.E2E_TEST_IDP }}
+          E2E_TEST_CLIENT_ID: ${{ secrets.E2E_TEST_CLIENT_ID }}
+          E2E_TEST_CLIENT_SECRET: ${{ secrets.E2E_TEST_CLIENT_SECRET }}
+          E2E_TEST_ENVIRONMENT: ${{ matrix.environment-name }}
+      - name: Add status check with successful test result
+        id: set-status-check-success
+        uses: octokit/request-action@v2
```

It looks like this is problematic. Should the patch version be still specified, and if so do you have an idea why it differs from other GH actions?

---

Well, our current directive is that we don't care for patch versions. Though, honestly, maybe we can get rid of this? idk

---

Apparently it should be
```suggestion
        uses: octokit/request-action@v2.x
```

---

I saw that in the docs, but I thought it was a hand-wavy example, and not the actual expected value ^^